### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by christulin**
## Trello Ticket
https://trello.com/c/4KejfGQW/24-set-up-connection-to-postgres-and-build-seeding-script

## Purpose of Pull Request
setting postgres schema, setting up connection between node and postgres, building seeding script.

### CODE REVIEW

File: database/aws-s3.js [hunk 1]
```diff
@@ -5,13 +5,34 @@ const AWS = require('aws-sdk');
...
+        }
+      });
+    } catch(err) {
+      console.log(err);
+    }
+  })
+}
+
 // Get all buckets
 const getBuckets = async () => {
```
Comment (gigicupp) [inline database/aws-s3.js:37]: just out of curiosity, when would this function be used?

File: database/pgtools.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+const pgtools = require('pgtools');
```
Comment (gigicupp) [inline database/pgtools.js]: Learned something :)

File: database/seed.js [hunk 3]
```diff
@@ -12,10 +12,13 @@ const seed = async () => {
 
   const photos = await listPhotos();
 
-  //loop 1000 times, in each loop, loop through photos url, get applicable data points, and seed db
+  // loop 1000 times, in each loop, loop through photos url, get applicable data points, and seed db
   photos.Contents.forEach(photo => {
-    const url = `https://sdc-airbnb-photos.s3.us-east-2.amazonaws.com/${photo.Key}`
-    console.log(url)
+    const storage_url = `https://sdc-airbnb-photos.s3.us-east-2.amazonaws.com/${photo.Key}`
+    const name = photo.Key.split('.')[0];
```
Comment (gigicupp) [inline database/seed.js]: Why is the .split() needed?

**Review by gigicupp — APPROVED** [review]
Hi Chris,
Looks good. Just added couple of inline questions out of curiosity.

File: database/aws-s3.js (hunk 1 above)
Comment (christulin) [reply database/aws-s3.js:37]: the `getBuckets` function is leftover from FEC, I don't have use for it.. will clean up this file soon.

File: database/seed.js (hunk 3 above)
Comment (christulin) [reply database/seed.js]: `photo.Key` returns a name that ends in .jpg.  I need that for url, but just using the file name without extension for the name.

**Pull request merged**